README.md

```diff
@@ -85,6 +85,12 @@ The region the AWS `bucket` is located in.
 
 *Default:* `us-east-1`
 
+### acl
+
+The ACL to apply to the objects.
+
+*Default:* `public-read`
```

Comment:
@flecno I can't see where you are setting the default value in the code. Can you please follow the conventions of the other default values in this file?